__tests__/client.test.ts

```diff
@@ -113,6 +113,24 @@ describe(`SPA client`, () => {
       expect(cacheMock.allKeys).toBeCalledTimes(1)
       expect(cacheMock.remove).toBeCalledTimes(2)
     })
+
+    it("should use NoCache strategy if browser hasn't SessionStorage", () => {
+      jest.spyOn(global.Storage.prototype, 'getItem').mockImplementation(() => {
+        throw new Error('Browser blocked access to sessionStorage')
+      })
+      const client = new FpjsClient({ loadOptions: getDefaultLoadOptions() })
+
+      expect(client.cacheLocation).toBe(CacheLocation.Memory)
+    })
+
+    it("should use NoCache strategy if browser hasn't LocalStorage", () => {
+      jest.spyOn(global.Storage.prototype, 'getItem').mockImplementation(() => {
```

Comment:
If this spied on the specific (local) storage, the test would have correctly revealed the bug in the implementation and would have failed. Isn't there any better way to spy on the specific storage instead? Perhaps using [this](https://stackoverflow.com/a/70971509/1692651)?